README.md

```diff
@@ -47,15 +47,20 @@ Prebuilt binaries are available to download at https://www.alluxio.io/download .
 Download and start an Alluxio master and a worker. More details can be found in [documentation](https://docs.alluxio.io/os/user/stable/en/deploy/Running-Alluxio-On-Docker.html).
 
 ```bash
-# launch a master
-$ docker run -d --net=host\
-    -v /mnt/data:/opt/alluxio/underFSStorage\
+# Launch the Alluxio master
+$ docker run -d \
+    -p 19999:19999 \
+    --net=alluxio_nw \
```

Comment:
why are we using this now?

Reply:
I copied the code snippet from docker doc:
https://docs.alluxio.io/os/user/2.0/en/deploy/Running-Alluxio-On-Docker.html

Is it not required?

Reply:
I thought we previously suggested using `--net=host` in our docker commands. Although it looks like everywhere now uses `alluxio_nw` instead. I'm just wondering if putting them on a separate docker network from the host has implications for first-time users being able to access the container

Reply:
I don't have a preference in which network to use. I suggest to be consistent with the docker guide to avoid unnecessary confusion on one using `--net=host` and the other using `--net=alluxio_nw`. For someone who are not familiar with docker, a difference like this will make me quite confused.

Reply:
well the issue I see with not using `--net=host` is that this doesn't work "out of the box". Running on my local machine I get 

```
zac-alluxio:golluxio zac-alluxio$ docker run -d \
>     -p 19999:19999 \
>     --net=alluxio_nw \
>     --name=alluxio-master \
>     -v ufs:/opt/alluxio/underFSStorage \
>     alluxio/alluxio master
Unable to find image 'alluxio/alluxio:latest' locally
latest: Pulling from alluxio/alluxio
4fe2ade4980c: Pull complete 
6fc58a8d4ae4: Pull complete 
fe815adf554b: Pull complete 
7a5be411d383: Pull complete 
a73bca27a3ba: Pull complete 
b7aedbbb5759: Pull complete 
2311b7bca327: Pull complete 
afb07e92fe12: Pull complete 
Digest: sha256:2e778463f8b7699dc60d576a1d03d7adc66141f9c2ded43e0a027467a8a21ea2
Status: Downloaded newer image for alluxio/alluxio:latest
73038f1522a7bba4586d88426260b977f7195f2cad80bf831953056236b1efca
docker: Error response from daemon: network alluxio_nw not found.
```

If we're worried about people inexperienced with docker getting started with this, it is not very good to have this kind of error

Reply:
ah, I didn't realize this. yes, then `--net=host` is the way to go. Will fix this

Reply:
The download page on the website also uses `--net=host`. May want to update that at some point too.

Reply:
I tested `--net=host` and got the following error with my local laptop

```
$ docker run -d   --net=host            --name=alluxio-master            -v ufs:/opt/alluxio/underFSStorage            alluxio/alluxio master
$$ docker logs 97586f501e2b
formatMaster is deprecated - use formatMasters instead
hostname: linuxkit-025000000001: Host not found
Formatting Alluxio Master @ 
2019-07-02 06:42:10,190 INFO  Format - Formatting master journal: /opt/alluxio-2.0.0/journal/
2019-07-02 06:42:10,212 ERROR Format - Failed to format
java.lang.RuntimeException: java.net.UnknownHostException: linuxkit-025000000001: linuxkit-025000000001: Name does not resolve
	at alluxio.util.network.NetworkAddressUtils.getLocalIpAddress(NetworkAddressUtils.java:499)
	at alluxio.util.network.NetworkAddressUtils.getLocalHostName(NetworkAddressUtils.java:421)
	at alluxio.util.network.NetworkAddressUtils.getConnectHost(NetworkAddressUtils.java:298)
	at alluxio.util.network.NetworkAddressUtils.getConnectAddress(NetworkAddressUtils.java:239)
	at alluxio.util.ConfigurationUtils.getMasterEmbeddedJournalAddresses(ConfigurationUtils.java:106)
	at alluxio.util.ConfigurationUtils.getEmbeddedJournalAddresses(ConfigurationUtils.java:90)
	at alluxio.master.journal.raft.RaftJournalConfiguration.defaults(RaftJournalConfiguration.java:54)
	at alluxio.master.journal.JournalSystem$Builder.build(JournalSystem.java:235)
	at alluxio.cli.Format.format(Format.java:118)
	at alluxio.cli.Format.main(Format.java:97)
Caused by: java.net.UnknownHostException: linuxkit-025000000001: linuxkit-025000000001: Name does not resolve
	at java.net.InetAddress.getLocalHost(InetAddress.java:1505)
	at alluxio.util.network.NetworkAddressUtils.getLocalIpAddress(NetworkAddressUtils.java:457)
	... 9 more
Caused by: java.net.UnknownHostException: linuxkit-025000000001: Name does not resolve
	at java.net.Inet4AddressImpl.lookupAllHostAddr(Native Method)
	at java.net.InetAddress$2.lookupAllHostAddr(InetAddress.java:928)
	at java.net.InetAddress.getAddressesFromNameService(InetAddress.java:1323)
	at java.net.InetAddress.getLocalHost(InetAddress.java:1500)
	... 10 more
```
And `--net=alluxio_nw` worked for me. Are you able to run with `--net=host`?

Reply:
Weird. Everything works fine for me running this command:

```bash
docker run -d \
           -p 19999:19999 \
           --net=host \
           --name=alluxio-master \
           -v ufs:/opt/alluxio/underFSStorage \
           alluxio/alluxio master
```

Reply:
Ok well I figured out it has the same issue for me. I would make sure that the commands for the `docker network create...`, `docker volume create...` are provided as well with this. (Same for the website)